### PR TITLE
Add customJS

### PR DESCRIPTION
**Pull request by oshliaer**
I didn't find a way to add a custom js file.

In my case I'm using [highlightjs] for highlighting Excels formulas. It works fine.

[highlightjs]: https://highlightjs.org/

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -43,6 +43,9 @@
 		{{ partial "footer" . }}
 	</div>
 <script async defer src="{{ "js/menu.js" | relURL }}"></script>
+{{ range .Site.Params.customJS -}}
+<script async defer src="{{ . | relURL }}"></script>
```
Comment (Vimux) [inline layouts/_default/baseof.html]: Hello, @oshliaer 

I do not think that `async defer` is suitable for all cases. Asynchronous loading for third-party scripts can break something.

Comment (oshliaer) [reply layouts/_default/baseof.html]: Can I add properties? The array of strings is not too informative.

Comment (oshliaer) [reply layouts/_default/baseof.html]: I've deleted this.

Comment (Vimux) [reply layouts/_default/baseof.html]: > Can I add properties? The array of strings is not too informative.

Sorry, I didn’t quite catch that. What do you mean?

**Comment by oshliaer** [comment]
> Can I add properties ...

I will do a new PR as soon as I am ready. Is it OK?

**Comment by Vimux** [comment]
> I will do a new PR as soon as I am ready. Is it OK?

Yes, take your time and do the new PR whenever you can. Just keep in mind that I didn’t quite catch what exactly do you mean about "properties".

For now, I merge this PR.

**Pull request merged**